sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -657,4 +658,22 @@ object AlluxioUtils extends Logging with Arm {
     }
     (location, mapIfReplacedPaths)
   }
+
+  // If reading large s3 files on a cluster with slower disks, skip using Alluxio.
+  def shouldReadDirectlyFromS3(rapidsConf: RapidsConf, pds: Seq[PartitionDirectory]): Boolean = {
+    if (!rapidsConf.enableAlluxioSlowDisk) {
+      return false
```

Comment:
prefer not to see returns, just make it an if else

Reply:
Done.